tests/System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryInfoTests.cs

```diff
@@ -444,6 +445,76 @@ public void MockDirectoryInfo_Exists_ShouldUpdateCachedDataOnRefresh()
             Assert.IsTrue(directoryInfo.Exists);
         }
 
+        [Test]
+        public void Directory_exists_after_creation()
+        {
+            // Arrange
+            var fileSystem = new MockFileSystem();
+            var directoryInfo = fileSystem.DirectoryInfo.FromDirectoryName(XFS.Path(@"c:\abc"));
+
+            // Act
+            directoryInfo.Create();
+
+            // Assert
+            Assert.IsTrue(directoryInfo.Exists);
+        }
+
+        [Test]
+        public void Directory_exists_after_creation_with_security()
```

Comment:
This can only run on Windows. See https://github.com/TestableIO/System.IO.Abstractions/blob/d7ba4dc9a47ec054730aba16fb1995ad77809cea/tests/System.IO.Abstractions.TestingHelpers.Tests/MockDirectoryInfoAccessControlTests.cs#L10 for how we handle this 🙃

Reply:
I'm confused. Are `MockDirectoryInfo` / `MockFileInfo` not platform-agnostic?

Reply:
The actual "magic" is [`WindowsOnlyAttribute`](https://github.com/TestableIO/System.IO.Abstractions/blob/d7ba4dc9a47ec054730aba16fb1995ad77809cea/tests/System.IO.Abstractions.TestingHelpers.Tests/WindowsOnlyAttribute.cs) which prevents the test/fixture they it is applied to from running on non-Windows operating systems. `WindowsSpecifics.AccessControlLists` is just a marker to declare (for us humans :smile:) why this specific test is Windows-only.